src/System.Runtime.TimeZoneData/System.Runtime.TimeZoneData.csproj

```diff
@@ -35,15 +36,19 @@
 
   <Target Name="UpdateData">
     <RemoveDir Directories="$(TimeZoneDataSrcDir)" />
-    <DownloadFile SourceUrl="https://data.iana.org/time-zones/tzdb-$(TimeZoneDataVersion)/%(Area.Identity)"
-                  DestinationFolder="$(TimeZoneDataIntermediateDir)" />
+    <Exec Command="wget -r -nH --cut-dirs=2 --no-parent --reject='index.html*' -P $(TimeZoneDataIntermediateDir) https://data.iana.org/time-zones/tzdb-2021a/" />
+    <Exec Command="touch README" WorkingDirectory="$(TimeZoneDataIntermediateDir)"/>
```

Comment:
Why is this necessary?

Reply:
We need to download the entire tzdb directory instead of just specific files. `make zic` doesn't work without a `README`, which is not (weirdly) included in the directory itself.

Reply:
...huh, is there any chance of PRing a fix for that upstream?

Reply:
It's possible we should switch to using the full tar.lz that contains now